config/anlaytics-settings.js

```diff
@@ -0,0 +1,4 @@
+exports.analyticsIds = {
+  procudtion: 'UA-55446531-10',
```

Comment:
```suggestion
  production: 'UA-55446531-10',
```

Reply:
🤦 I need to use grammerly on my vscode at some point